### DISCUSSION TITLE
libsdl2 dependencies

### DISCUSSION THREAD

**Discussion by fartboxmk2** [GameServerManagers/LinuxGSM#3743] [Q&A]
When running `./inssserver install`  I get this message
`Warning! Missing dependencies: libsdl2-2.0-0:i386`
`Failure! Missing dependencies required to run SteamCMD.
`
so I do apt install libsdl2-2.0-0:i386 and it wants to install the following packages;

`libasound2:i386 libasound2-data libasyncns0:i386 libbsd0:i386 libcap2:i386 libdbus-1-3:i386 libffi6:i386 libflac8:i386 libgcrypt20:i386 libgpg-error-l10n libgpg-error0:i386
  libice6:i386 liblz4-1:i386 liblzma5:i386 libogg0:i386 libpulse0:i386 libsm6:i386 libsndfile1:i386 libsndio7.0:i386 libsystemd0:i386 libuuid1:i386 libvorbis0a:i386
  libvorbisenc2:i386 libwayland-client0:i386 libwayland-cursor0:i386 libwayland-egl1:i386 libwrap0:i386 libx11-6:i386 libx11-data libx11-xcb1:i386 libxau6:i386 libxcb1:i386
  libxcursor1:i386 libxdmcp6:i386 libxext6:i386 libxfixes3:i386 libxi6:i386 libxinerama1:i386 libxkbcommon0:i386 libxrandr2:i386 libxrender1:i386 libxss1:i386 libxtst6:i386
  libxxf86vm1:i386 uuid-runtime x11-common xkb-data
`

is it normal to install a bunch of desktop X11/wayland packages? I didn't have to install these for my mordhau server that also uses steamcmd.

Edit: This is on Debian 10

**Comment by johnoclockdk** [comment]
have you installed dependencies
 https://linuxgsm.com/servers/inssserver/#v-pills-dependencies

**Reply by fartboxmk2** [reply]
Yes. But on that page the Debian install doesn't list libsdl2 as a dependency, but when I run ./install inssserver it is listed a missing dependency. So I wasn't sure if this is a bug on the install script because I didn't have to install libsdl2 for my other steamCMD(mordhau) install.

**Reply by johnoclockdk** [reply]
then is the website missing it
